app_feup/lib/model/entities/exam.dart

```diff
@@ -75,6 +75,15 @@ class Exam {
     };
   }
 
+  bool beforeEndTime() {
```

Comment:
Naming could be clearer. Maybe "hasEnded()"?

Reply:
done!